src/controlplane/client/discovery.go

```diff
@@ -35,28 +32,55 @@ func (i invalidTLSConfig) Error() string {
 	return i.message
 }
 
-type authenticationMethod string
+type AuthenticationMethod string
 
 const (
-	none           authenticationMethod = "None (http)"
-	mTLS           authenticationMethod = "Mutual TLS"
-	serviceAccount authenticationMethod = "Service account (Bearer token)"
+	None           AuthenticationMethod = "None (http)"
+	MTLS           AuthenticationMethod = "Mutual TLS"
```

Comment:
should we comment these?

Reply:
mmm this will probably gone when refactored the client